File: bash/template/java/Testament.java

package $(padded_ordinal)_$(snaked_problem);

import org.junit.Test;
import org.junit.Assert;

public class Testament {

    private Solution solution = new Solution ();

    @Test void
    solution_works () {
        ${boxing} expected = new ${boxing} ();
        ${boxing} actual   = solution.${method} (...);
        Assert.assertEquals (expected, actual);
    }

}

File: bash/template/javascript/testament.js

let assert = require ("assert");
let Solution = require ("Solution");
let tests = [
    solutionWorks
];

function solutionWorks () {
    let original = null;
    let expected = null;
    let actual = Solution.${function} (original);
    assert.equal (expected, actual);
}

for (var i = 0; i < tests.length; i++) tests [i] ();
